### ISSUE TITLE
Add rule protecting against vague, generic labels

### ISSUE THREAD

**Issue by kzhou314** [github/accessibility-scanner-alt-text-plugin#8]
Add a vague alt text check to the alt-text plugin. The rule should flags <img> elements whose alt text consists only of generic, non-descriptive words or phrases (e.g. "image", "a photo of", "screenshot").

This kind of alt text satisfies the presence check but provides no useful information, violating [WCAG 1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content).

Satisfies an acceptance criteria of github/accessibility/issues/10633.
